nomos-services/storage/src/backends/mod.rs

```diff
@@ -45,6 +45,7 @@ pub trait StorageBackend: Sized {
     fn new(config: Self::Settings) -> Result<Self, Self::Error>;
     async fn store(&mut self, key: Bytes, value: Bytes) -> Result<(), Self::Error>;
     async fn load(&mut self, key: &[u8]) -> Result<Option<Bytes>, Self::Error>;
+    async fn load_prefix(&mut self, prefix: &[u8]) -> Result<Vec<Bytes>, Self::Error>;
```

Comment:
What does this do? Maybe is up to the user to actually compute this? it feels bad to add something that is not common to other storages. Just trying to understand it.

Reply:
In key value store (sled, rocksdb etc) there's an ability to load values by incomplete key. For example `/da/verified/blob_id` in this case would return all blobs from different subnetworks that the node has (keys in that case would be `/da/verified/blob_id1` .. `/da/verifier/blob_idN`. I think this is useful feature of such databases, and could also be implemented in relational dbs, although the name `LoadPrefix` could be improved. In case of sled and rocksdb this name reflects what it does directly.